src/load/azext_load/data_plane/load_test/help.py

```diff
@@ -32,6 +32,10 @@
     - name: Create a test for a private endpoint in a Virtual Network with split CSV option enabled.
       text: |
         az load test create --test-id sample-test-id --load-test-resource sample-alt-resource --resource-group sample-rg --display-name "Sample Name" --subnet-id "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-rg/providers/Microsoft.Network/virtualNetworks/SampleVMVNET/subnets/SampleVMSubnet" --split-csv true
+    - name: Create a test with custom defined autostop criteria or disable autostop for a test.
+      text: |
+        az load test create --test-id sample-test-id --load-test-resource sample-alt-resource --resource-group sample-rg --display-name "Sample Name" --autostop-error-rate 80.5 --autostop-error-rate-time-window 120
```

Comment:
should we add the unit of time window ? 120 s ?
--autostop-error-rate-time-window-in-sec ??

Reply:
Yeah, I agree. The name is getting too long. Help + docs should we enough